machine/wnc_x86_64/INSTALL

```diff
@@ -0,0 +1,36 @@
+=============================================
+Installing ONIE on WNC x86_64
```

Comment:
The directory layout for this machine type will not work out.

It is recommended to use the following format for the machine directory:

`machine/<vendor>/<vendor>_<model>`

See the Accton AS6712_32X as an example:
https://github.com/opencomputeproject/onie/tree/master/machine/accton/accton_as6712_32x